manual/idempotence/README.md

```diff
@@ -0,0 +1,132 @@
+## Query idempotence
+
+A query is *idempotent* if it can be applied multiple times without changing the result of the initial application. For
```

Comment:
Hrm, think I've changed my mind about this.  This might be more of a matter of perspective, the client doesn't get a completed Future until the operation succeeds, so even if it fails and retries and there end up being multiple mutations, the user knows that the operation completed.  Where it matters is for LWTs as you mentioned in the documentation.